src/styles/style-helpers.ts

```diff
@@ -29,27 +28,24 @@ export const getTheme = (args: ThemeArgs): Theme =>
 export const shadowSingle = (args: ThemeArgs): SerializedStyles => {
   const theme = getTheme(args);
   return css`
-    box-shadow: 0 0 0 1px ${transparentize(0.98, theme.colors.shadow)},
-      0 0 1px 0 ${transparentize(0.94, theme.colors.shadow)},
-      0 2px 2px 0 ${transparentize(0.94, theme.colors.shadow)};
+    box-shadow: 0 0 0 1px ${theme.colors.shadow},
```

Comment:
Why are we changing this in this PR?

Reply:
I needed to get rid of the `transparentize` function from _polished_.